src/main/groovy/Limelight.groovy

```diff
@@ -34,10 +34,11 @@ class Limelight {
 
     def run(config) {
         def id = config['username']
+        def shortname = config['shortname']?config['shortname'] : id
```

Comment:
I wasn't sure what would happen for already saved Limelight configs. I assumed they would be loaded with a 'username' but no 'shortname.' Instead of populating a shortname I decided to just set the shortname to be the username, since for any working feeds they will have had to have been the same